workload/traefik.yaml

```diff
@@ -230,10 +230,10 @@ spec:
         # PRODUCTION READINESS CHANGE REQUIRED
         # This image should be sourced from a non-public container registry, such as the
         # one deployed along side of this reference implementation.
-        # az acr import --source docker.io/library/traefik:v2.10.4-n <your-acr-instance-name>
+        # az acr import --source docker.io/library/traefik:v2.10.7 -n <your-acr-instance-name>
         # and then set this to
-        # image: <your-acr-instance-name>.azurecr.io/library/traefik:v2.10.4
-      - image: docker.io/library/traefik:v2.10.4
+        # image: <your-acr-instance-name>.azurecr.io/library/traefik:v2.10.7
+      - image: docker.io/library/traefik:v2.10.7
```

Comment:
You need to update this on **09-secret-management-and-ingress-controller.md** as well.

Reply:
Thanks - great find